LICENSE

```diff
@@ -0,0 +1,8 @@
+This work is licensed under the PostgreSQL License, as given at
+http://www.postgresql.org/about/licence/	
+
```

Comment:
(Somehow my comment has been dropped, trying again)

Please include a copyright statement here (see https://github.com/percona/pg_stat_monitor/blob/master/LICENSE as an example):

```
Copyright (c) 2021 Percona LLC and/or its affiliates
```

Also please replace `THE UNIVERSITY OF CALIFORNIA` with `THE COPYRIGHT HOLDER`, as this documentation is copyrighted by Percona.

Reply:
Fixed